Application/ResetSystem/ResetSystem.c

```diff
@@ -42,6 +46,39 @@ UefiMain (
     Mode = L"ColdReset";
   }
 
+  if (StrCmp (Mode, L"Firmware") == 0) {
+    DEBUG ((DEBUG_INFO, "OCRST: Entering firmware...\n"));
+    DataSize = sizeof (OsIndications);
+    Status = gRT->GetVariable (
+      EFI_OS_INDICATIONS_SUPPORT_VARIABLE_NAME, &gEfiGlobalVariableGuid,
+      &Attr, &DataSize, &OsIndications
+      );
+    if (!EFI_ERROR (Status)) {
+      if ((OsIndications & EFI_OS_INDICATIONS_BOOT_TO_FW_UI) != 0) {
+        DataSize = sizeof (OsIndications);
+        Status = gRT->GetVariable (
+          EFI_OS_INDICATIONS_VARIABLE_NAME, &gEfiGlobalVariableGuid,
```

Comment:
Could you please reformat this and the other calls to GetVariable/SetVariable with the usual EDK II codestyle (every argument on a separate line).

Reply:
Reformatted according to EDK II C Coding Standards Specification 2.1 5.2.2.4. The change has been squashed into previous commit.